cogs/kiem_tra_api.py

```
@@ -0,0 +1,63 @@
+import discord
+
+from discord import app_commands
+from discord.ext import commands
+
+from utils.embed import error_embed, create_embed, EmbedColor
+from utils.embed import error_embed, disabled_command_embed
```

Ruff (0.13.1)

Redefinition of unused `error_embed` from line 6

Remove definition: `error_embed`

(F811)